docs/general/serializers.md

```diff
@@ -217,7 +217,15 @@ The object being serialized.
 
 #### #root
 
-PR please :)
+Resource root which is included in `JSON` adapter. As you can see at [Adapters Document](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/adapters.md), `Attribute` adapter (default) and `JSON API` adapter dose not include root at top level.
+By default, the resource root comes from the class name of the serialized class.
+
+There are some ways to specify root like below:
+* [Overriding the root key](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/rendering.md#overriding-the-root-key)
+* [Setting `type`](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/serializers.md#type)
+* Passing an argument like `root: 'specific_name'` to the serializer's initialization.
```

Comment:
I'd rather change it a bit to something like:

```diff
-* Passing an argument like `root: 'specific_name'` to the serializer's initialization.
+* Specifying the `root` option, e.g. `root: 'specific_name'`, during the serializer's initialization:
+
+```ruby
+ActiveModelSerializers::SerializableResource.new(foo, root: 'bar')
+```
```